scripts/init-development-environment.js

```diff
@@ -0,0 +1,47 @@
+const force = process.argv.length > 2 && process.argv.slice(2).find(arg => arg === '-f' || arg === '--force');
+const empty = process.argv.length > 2 && process.argv.slice(2).find(arg => arg === '--empty');
+
+const environmentLocalPath = 'src/environments/environment.development.ts';
```

Comment:
Probably only a small thing, but since the whole thing got renamed, we can also rename that variable

Reply:
"historic footprint" 😅